scripts/package-template/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "suite-package-template",
+    "version": "1.0.0",
+    "main": "src/index",
+    "license": "See LICENSE.md in repo root",
+    "private": true,
+    "sideEffects": false,
+    "type": "module",
+    "scripts": {
```

Comment:
doesn't it require a "build" script?

Reply:
no this generator is just for "private" packages that will be used in suite, no need to build them

if you want to create some package that will be published (like packages related to connect), then you will have to do some changes like build script and few more

Reply:
I will probably add some template for these public "builded" packages later